### ISSUE TITLE
[Request] Setup A Repository

### ISSUE THREAD

**Issue by seniorm0ment** [osmc/skin.osmc#304]
Manually installing the OSMC skin, and keeping updated on systems like Android TV is a bit tedious. It would be really nice if it could be added to a repository to simply keep up to date. 
I looked on the github pages, and the website and saw no mention of a repository.

**Comment by samnazarko** [comment]
I don't believe that our skin can be added to the Kodi repository, because it depends on additional scripts which break the rules.

**Comment by seniorm0ment** [comment]
Could it not go in an unofficial repo, or consider setting up a repo you guys host that users can add?

**Comment by samnazarko** [comment]
I don't see any benefit of this for OSMC.
You'd still also need to manually add a new repo for every significant Kodi version change.

Maybe it's something we can look in to in the future.

**Comment by Ch1llb0** [comment]
We've discussed this in the team before. Pushing the skin upstream to the Kodi repo is not really an option for us as we're offering two versions of the skin (both the normal and the scope skin) and Kodi wouldn't accept both. Two versions of one skin are not allowed, if they aren't significantly different, it wouldn't make sense for us to push only one of the two (even a mod versions of skins have been an issue before). We do however have good reasons to maintain two versions of the skin and will continue doing so for now.
An OSMC Kodi repo has been an idea we've discussed as well, but it hasn't been feasible until now as it would, as @samnazarko says, only contain our skin(s). We do publish [releases of the skin(s) here on GitHub](https://github.com/osmc/skin.osmc/releases) every time we also push an update to our OSMC users. Although automatic updates after adding an OSMC repo to your Kodi setup would be nice, it would increase the support burdon for us significantly. But we'll keep an open mind for the future. Atm, this won't change though, I'm afraid. I hope you'll use and enjoy our OSMC Skin nonetheless! 👍🏻
